src/Search_Replace_Command.php

```diff
@@ -649,6 +649,30 @@ private static function esc_sql_ident( $idents ) {
 		return array_map( $backtick, $idents );
 	}
 
+	/**
+	 * Puts MySQL string values in single quotes, to avoid them being interpreted as column names.
+	 *
+	 * @param string|array $values A single value or an array of values.
+	 * @return string|array A quoted string if given a string, or an array of quoted strings if given an array of strings.
+	 */
+	private static function esc_sql_value( $values ) {
+		$quote = function ( $v ) {
+			// Don't quote numeric values to MySQL's implicit type conversion.
+			if ( is_numeric( $v ) ) {
```

Comment:
I'd prefer to only not quote decimal integer values rather than general numerics as PHP's definition may or may not match MySQL's definition. Highly unlikely and haven't been able to make it fail (!) but maybe just:
```
if ( '' !== $v && strlen( $v ) === strspn( $v, '0123456789' ) ) {
```
Tables with anything other than ints or strings as primary keys would be very rare anyway.

Reply:
Agree, but used a different check.